script/plugin_webpack_directories.rb

```diff
@@ -27,7 +27,7 @@ def log_plugin(plugin_name)
 def entry_name(plugin_name, entry)
   prefix = File.basename(entry).gsub(/[_]?index.js$/, '')
   if prefix != ''
-    "#{plugin_name}_#{prefix}"
+    "#{plugin_name}:#{prefix}"
```

Comment:
I changed this to ':' to prevent plugins that start with 'foreman_' from being recompiled when just 'foreman' is requested, this may never actually be an issue, but it seemed like something that may cause confusion in the future.

Reply:
meaning, this will require a katello change too

Reply:
i also assume that no other plugins are doing this

Reply:
reference: https://github.com/Katello/katello/pull/7930